Main.java

```diff
@@ -1,21 +1,35 @@
 class Main {
   public static void main(String[] args) {
    // kindly use appropriate data types for your declaration
-   // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor is to pay and assign values to them
-   
+   // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor has paid and assign values to them
+   byte days_default=4;
+   float amount_perday=400;
```

Comment:
Your amount_perday surely cannot be float as days cannot be fractions nor decimal